source/opt/function.h

```diff
@@ -79,6 +79,10 @@ class Function {
   // Saves the given function end instruction.
   inline void SetFunctionEnd(std::unique_ptr<Instruction> end_inst);
 
+  // Add a non-semantic instruction that succeeds this function in the module.
```

Comment:
Should you say where the new instruction will be relative to the previously existing non-semantic instructions succeeding this function?  Does it come before all of them or after all of them?